README.md

```diff
@@ -6,7 +6,7 @@ Its purpose is to allow the public to track changes to the Factorio **Lua protot
 
 See https://github.com/wube/factorio-data/releases for past versions.
 Compare versions using git on the command line, or by using magic github comparison urls, 
-eg: https://github.com/wube/factorio-data/compare/0.14.23...0.15.40
+eg: https://github.com/wube/factorio-data/compare/0.17.79...0.18.31
```

Comment:
Hey, sorry that this review is so delayed! Would you mind bumping these versions to something more current? Like `1.1.50` and `1.1.56` for example?

I'll merge or close this PR within the week; either way the example versions will be bumped.